index.html

```diff
@@ -1625,7 +1620,14 @@
           src="/images/Group_4.png"
           alt="Group 4"
           class="q191"
+          usemap="#workmap"
         >
```

Comment:
- [ ] Kindly, check the way you are calling images in your HTML because your project has some bugs regarding that issue, please fix it. For more check the below images.

![first](https://user-images.githubusercontent.com/55635977/125177152-c8537e80-e1d9-11eb-9321-f4283cc4870e.png)
![second](https://user-images.githubusercontent.com/55635977/125177157-cc7f9c00-e1d9-11eb-9af2-fd14f0645037.png)

Reply:
Fixed! Thanks, man!!